R/DittoScatterPlot.R

```diff
@@ -25,7 +26,7 @@
 #' @param cells.use String vector of cells'/samples' names OR an integer vector specifying the indices of cells/samples which should be included.
 #' 
 #' Alternatively, a Logical vector, the same length as the number of cells in the object, which sets which cells to include.
-#' @param show.others Logical. TRUE by default, whether other cells should be shown in the background in light gray.
+#' @param show.others Logical. FALSE by default, whether other cells should be shown in the background in light gray.
```

Comment:
This should stay as TRUE, no?

Reply:
It's FALSE by default in the call. TRUE by default for dittoDimPlot.

Reply:
this was not on purpose... Not sure whether it's better for the default to be changed instead... But I'll do that on my own another time if needed.

This is fine then. Thanks for the catch!